contents/ai/ml.md

```diff
@@ -103,7 +103,7 @@ Data is broadly split into 2 categories, *structured* and *unstructured*.
 
 The distinction between the types of data is important as ML algorithms are not always compatible with both data types. For example, [Decision trees](https://en.wikipedia.org/wiki/Decision_tree) cannot be used for unstructured data. Data must be modified to a structured form for use with decision tree related algorithms.
 
-## How ML is Applied
+## How Machine Learning is Applied?
```

Comment:
If we want to phrase the heading as a question, perhaps we should use `How is Machine Learning Applied?` instead?

Reply:
Agreed.